### PR TITLE
migrate Mongoose to Prisma

### PR DESCRIPTION
**Pull request by OfficialSirH**
remove (currently comment out) all usage of mongoose(MongoDB) and replace it with Prisma(PostgreSQL)

### CODE REVIEW

**Comment by amitojsingh366** [comment]
it is vary nice, prisma >>> mongoose

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
+#prisma migrations
+prisma/migrations/[0-9]*_[a-zA-Z]*/
```
Comment (amitojsingh366) [inline .gitignore:2]: sirh wait what the, why are you gitignoring migrations?

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (ToasterSticks) [reply .gitignore:2]: Oh no!

Comment (amitojsingh366) [reply .gitignore]: > > sirh wait what the, why are you gitignoring migrations?
> 
> mmlol

i wasn't joking 🙃

**Comment by OfficialSirH** [comment]
@amitojsingh366 there ya go 🙃

**Comment by amitojsingh366** [comment]
> @amitojsingh366 there ya go 🙃

nice 🙃

**Pull request merged**